CONTRIBUTING.md

```diff
@@ -58,33 +58,30 @@ You can now use the shortcut to run the program from the repository. Running the
 * While in the Tree tab, holding `Alt` also highlights nodes that have unrecognised modifiers.
 * Holding `Ctrl` while launching the program will rebuild the mod cache.
 
-Note that the updates system is disabled in Dev Mode, so you must update manually.
+Note that automatic updates are disabled in Dev Mode, so you must update manually.
 
 #### Keeping your fork up to date
 
 Note: This tutorial assumes that you are already familiar with Git and basic command line tools.
 
-Note: If you've configured a remote already, you can skip ahead to step 6.
+Note: If you've configured a remote already, you can skip ahead to step 3.
 
-1. Check your current remote repositories.
-
-       git remote -v
-2. Add a new remote repository and name it `upstream`.
+1. Add a new remote repository and name it `upstream`.
 
        git remote add upstream https://github.com/PathOfBuildingCommunity/PathOfBuilding.git
-3. Verify that adding the remote worked by running the last command again.
+2. Verify that adding the remote worked by running the last command again.
```

Comment:
```suggestion
2. Verify that adding the remote worked.
```
Running the last command again results in `fatal: remote upstream already exists.`